scipy/stats/_multivariate.py

```diff
@@ -982,12 +993,12 @@ def _process_parameters(self, mean, rowcov, colcov):
         if mean is not None:
             if meanshape[0] != numrows:
                 raise ValueError("Arrays `mean` and `rowcov` must have the"
-                                  "same number of rows.")
+                                 "same number of rows.")
             if meanshape[1] != numcols:
                 raise ValueError("Arrays `mean` and `colcov` must have the"
-                                  "same number of columns.")
+                                 "same number of columns.")
```

Comment:
Might be worth adding a space at end of first line or at start of second line here & above so that `the` and `same` don't get smushed together in the raised error message.

Reply:
I hadn't noticed that; will do.